docs/self-managed/operational-guides/production-guide.md

```diff
@@ -0,0 +1,572 @@
+---
+id: production-guide
+title: "Helm Chart Production Guide"
+sidebar_label: "Helm Chart Production Guide"
+description: "Learn how to set up the helm chart in a production setting."
+---
+
+## Overview
+
+This guide provides a simplified and streamlined architecture for core Camunda 8 applications using the Camunda Helm Chart. This setup minimizes complexity while offering a reliable foundation for most production use cases.
+
+The goal of this guide is to give you a **scenario-based, production focused, step-by-step guide** for setting up the Camunda Helm Chart. We will use AWS as a reference here but the guide would work with other supported cloud providers and their comparable services as well. By following this guide, you will be familiar with all of the necessary requirements for having a production ready Camunda Helm Chart.
+
+## Prerequisites
+
+Before proceeding with the setup, ensure the following requirements are met:
+
+- **Kubernetes Cluster**: A functioning Kubernetes cluster with kubectl access.
+- **Helm**: Helm CLI installed
+- **DNS Configuration**: Access to configure DNS for your domain to point to the Kubernetes cluster ingress.
+- **TLS Certificates**: Obtain valid X.509 certificates for your domain from a trusted Certificate Authority.
+- **External Dependencies**: Provision the following external dependencies:
+  - **Amazon Aurora PostgreSQL**: For persistent data storage required for the Web Modeler component.
+  - **Amazon OpenSearch**: For indexing and analytics.
+  - **AWS Simple Active Directory**: For authentication and authorization.
+- **NGINX Ingress Controller**: Ensure the NGINX ingress controller is set up in the cluster.
+- **AWS OpenSearch Snapshot Repository** - This will be a place to store the backups of the Camunda cluster. This repository must be configured with OpenSearch to take backups.
```

Comment:
just looking for `snapshot` in the doc, I guess it's not further covered.
Could we link to something upstream from AWS on the topic?

Reply:
I will reference this document: https://docs.aws.amazon.com/opensearch-service/latest/developerguide/managedomains-snapshot-registerdirectory.html